examples/nodejs/token-vending-machine/lambda/config.ts

```diff
@@ -0,0 +1,44 @@
+import {AllDataReadWrite, TopicRole, CacheRole, ExpiresIn, TokenScope, AllTopics, AllCaches} from '@gomomento/sdk';
+
+/**
+ * Set the scope of permissions for your tokens. 
+ * 
+ * AllDataReadWrite provides read and write permissions to all of your caches:
+ *    export const tokenPermissions: TokenScope =  AllDataReadWrite;
+ * 
+ * You may also provide a bespoke list of permissions for each cache and topic that you have:
+ *    export const tokenPermissions: TokenScope =  {
+ *      permissions: [
+ *        {
+ *          role: CacheRole.ReadWrite | CacheRole.ReadOnly, 
+ *          cache: AllCaches | "your-cache-name"
+ *        },
+ *        {
+ *          role: TopicRole.PublishSubscribe | TopicRole.SubscribeOnly, 
+ *          cache: AllCaches | "your-cache-name",
+ *          topic: AllTopics | "your-topic-name"
+ *        }
+ *      ]
+ *    };
+ * 
+ * More information here: https://github.com/momentohq/client-sdk-javascript/blob/62a34741059cb08c8dfff4e7011be29facab8d80/packages/core/src/auth/tokens/token-scope.ts
```

Comment:
i love that you used the permalink for the gh link instead of HEAD :)

but I think for these readme's let's try to steer people toward the dev docs when we can.  so for this one we can use:

https://docs.momentohq.com/develop/sdks/web#credentials-for-browsers